test/MainEvent.cxx

```diff
@@ -131,9 +131,12 @@ int main(int argc, char **argv)
    Int_t branchStyle = 1; //new style by default
    if (split < 0) {branchStyle = 0; split = -1-split;}
 
+   ROOT::EnableImplicitMT(4);
```

Comment:
I disagree ;). I think it is a useful new testing swtich ... it just need to be made conditional.  thanks.